control.lua

```diff
@@ -78,6 +87,13 @@ local function onTick(event)
   end
 end
 
+
```

Comment:
Interfaces to catch events from AAI_Programmable_Vehicles. Receives AAI data table then calls "adapter" with entity parameter.